### DISCUSSION TITLE
Missing test-options in v5 documentation

### DISCUSSION THREAD

**Discussion by poifir** [vmware/clarity#6011] [Ideas] [answered]
Starting with the Clarity v5 documentation all options for manual tests within the documentation got removed (and videos of the components got added as replacement).

The v4 docu still allows to test the different components, but of course using the old docu is not a future proof option.

The videos in the v5 documentation are imho only an addition, but cannot be treated as replacement for the test options.

Please re-integrate test-options for all components into the v5 documentation as it has been the case in previous releases.

**Comment by Jinnie** [comment, marked as answer]
We're working on adding a v5 documentation that will be similar to the v4 docs.
